### PR TITLE
Feature/order to orders

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
## PR Overview
**This PR will address the following Issue/Feature:**
- https://github.com/fivetran/dbt_recharge_source/issues/11

**This PR will result in the following new package version:**

- v0.2.0 (assuming source is breaking)

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [ ] ~dbt run (if incremental models are present) && dbt test~

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked, tagged, and properly assigned
- [x] All necessary documentation and version upgrades have been applied
- [x] docs were regenerated (unless this PR does not include any code or yml updates)
  - [x] will regen after approval
- [x] BuildKite integration tests are passing
- [x] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

- Validation done in source. Only change in transform is pointing to the updated source.

### If you had to summarize this PR in an emoji, which would it be?
 
🍟

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,7 +75,14 @@ vars:
 ``` 
 
 ## (Optional) Step 5: Additional configurations
-<details><summary>Expand for configurations</summary>
+<details open><summary>Expand for configurations</summary>
+
+### Leveraging `orders` vs `orders` source
```
Comment (fivetran-joemarkiewicz) [inline README.md:80]: Same suggestion that I think you meant for this to be the two different names
```suggestion
### Leveraging `orders` vs `order` source
```

File: README.md [hunk 2]
```diff
@@ -75,7 +75,14 @@ vars:
...
+For Fivetran Recharge connectors created on or after June 18, 2024, the `ORDER` source table has been renamed to `ORDERS`. The package will default to use the `ORDERS` table if it exists and then `ORDER` if not. If you have both versions but wish to use the `ORDER` table instead, you can set the variable `recharge__using_orders` to false in your `dbt_project.yml` file.
```
Comment (fivetran-joemarkiewicz) [inline README.md:81]: Same comment to add the link to the release notes for the connector here.

File: integration_tests/dbt_project.yml [hunk 3]
```diff
@@ -66,7 +65,7 @@ seeds:
         APPLIES_TO_ID: "{{ 'INT64' if target.type == 'bigquery' else 'bigint' }}"
         ENDS_AT: "timestamp"
         STARTS_AT: "timestamp"
-    order_data:
+    orders_data:
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/dbt_project.yml:68]: Same comment from the source to get your thoughts on if we should still test for the `order` name in our integration tests

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
@fivetran-catfritz same notes from the source PR - Let me know once updates are applied and I can re-review. Overall this looks great!

File: integration_tests/dbt_project.yml (hunk 3 above)
Comment (fivetran-catfritz) [reply integration_tests/dbt_project.yml:74]: Agreed--added!

File: README.md (hunk 1 above)
Comment (fivetran-catfritz) [reply README.md]: Thank you--updated!

File: README.md (hunk 2 above)
Comment (fivetran-catfritz) [reply README.md]: updated!

File: packages.yml [hunk 4]
```diff
@@ -1,3 +1,6 @@
 packages:
- - package: fivetran/recharge_source
-   version: [">=0.1.0", "<0.2.0"]
+#  - package: fivetran/recharge_source
+#    version: [">=0.2.0", "<0.3.0"]
+- git: https://github.com/fivetran/dbt_recharge_source.git
+  revision: feature/order-to-orders
+  warn-unpinned: false
```
Comment (fivetran-catfritz) [inline packages.yml:6]:
```suggestion
 - package: fivetran/recharge_source
   version: [">=0.2.0", "<0.3.0"]
```

**Review by fivetran-catfritz — COMMENTED** [review]
@fivetran-joemarkiewicz Thanks! I updated here as well.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM after docs are regenerated!

File: packages.yml (hunk 4 above)
Comment (fivetran-jamie) [reply packages.yml:6]: 👀

**Review by fivetran-jamie — APPROVED** [review]
lgtm, just gotta update the packages.yml!

**Pull request merged**